tests/foreman/performance/test_candlepin_concurrent_delete.py

```diff
@@ -0,0 +1,129 @@
+"""Test class for concurrent subscription deletion"""
+from datetime import date
+from robottelo.common import conf, ssh
+
+from robottelo.performance.perf_concurrent_test import ConcurrentTestCase
+
+
+class ConcurrentDeleteTestCase(ConcurrentTestCase):
+    """Concurrent Deleting subscriptions tests"""
+
+    @classmethod
+    def setUpClass(cls):
+        super(ConcurrentDeleteTestCase, cls).setUpClass()
+        # parameters for concurrent activation key test
+        cls.savepoint = conf.properties[
+            'performance.test.savepoint2_enabled_repos']
+        cls.raw_file_name = conf.properties[
+            'performance.csv.raw_ak_file_name']
+        cls.stat_file_name = conf.properties[
+            'performance.csv.stat_ak_file_name']
+
+        # add date to csv files names
+        today = date.today()
+        cls.raw_file_name = '{0}-{1}'.format(
+            today.strftime('%Y%m%d'), cls.raw_file_name)
+        cls.stat_file_name = '{0}-{1}'.format(
+            today.strftime('%Y%m%d'), cls.stat_file_name)
+
+    def setUp(self):
+        super(ConcurrentTestCase, self).setUp()
+
+        # Restore database before concurrent deleting subscribed systems
+        self._restore_from_savepoint(self.savepoint)
```

Comment:
You are calling `self._restore_from_savepoint(self.savepoint)` in every class that inherit from ConcurrentTestCase, you can move this to the setUp of the base class to avoid repeating this for every subclass.